service/controller/v11/templates/cloudformation/guest/instance.go

```diff
@@ -25,6 +25,9 @@ const Instance = `{{define "instance"}}
       Size: 50
       VolumeType: gp2
       AvailabilityZone: !GetAtt {{ .Instance.Master.Instance.ResourceName }}.AvailabilityZone
+      Tags:
+      - Key: Name
+        Value: {{ .Instance.Master.DockerVolume.Name }}
```

Comment:
This is the important reference we need to have, otherwhise we are not able to list the correct volume in order to detach it.